api_testing/apis/orders_api.py

```diff
@@ -39,7 +39,9 @@ def get_suborder_by_order_id(self, id=0):
         """
         """
         api = '{}{}{}'.format(self.url, self.END_POINTS['orders_api']['get_suborder'], str(id)+'&deleted=0')
-        return api, {}
+        _payload = {}
+        return api, _payload
+        #return api, {}
```

Comment:
this function works, why you updated  it

Reply:
Done